src/index.d.ts

```diff
@@ -32,4 +32,7 @@ interface Window {
 }
 
 declare const __PLAYROOM_GLOBAL__CONFIG__: InternalPlayroomConfig;
-declare const __PLAYROOM_GLOBAL__STATIC_TYPES__: any;
+declare const __PLAYROOM_GLOBAL__STATIC_TYPES__: Record<
+  string,
+  Record<string, unknown>
+>;
```

Comment:
This type isn't as precise as it could be. I'm unsure of the exact type it _should_ be, but it's better than `any`.

Reply:
I've made it more precise in https://github.com/seek-oss/playroom/pull/381/commits/7354aee2c8e8fa87bebaa211352c4bd40847fc00. This won't change anything at runtime of course, so the types aren't really that important, but if we ever convert more stuff to typescript it could be useful.